app/src/test/contrib/components/ChooseAction/ChooseActionPageTests.tsx

```diff
@@ -14,24 +14,18 @@ import {chooseActionPageActionCreators} from "../../../../main/contrib/actions/p
 import {ChooseActionContent} from "../../../../main/contrib/components/ChooseAction/ChooseActionContent";
 import {mockMatch} from "../../../mocks/mocks";
 
-describe("Choose Action Page Component", () => {
+describe("Choose Action Page", () => {
 
     const sandbox = new Sandbox();
     afterEach(() => sandbox.restore());
 
-    it("renders component on connect level", () => {
```

Comment:
I deleted this as we already check that the page action creator's `onLoad` method is called in the next test. Similarly for the other test suites.